### DISCUSSION TITLE
URL query by gene ID in linear genome and linear synteny view

### DISCUSSION THREAD

**Discussion by andreaminio** [GMOD/jbrowse-components#3148] [General] [answered]
Hi everyone,

is there a way to access a gene location on a browser by query URL instead of the location, like it was possible on JBrowse ver.1?

Andrea

**Comment by cmdcolin** [comment]
Currently there is not, but this would be useful to have. I will transfer this discussion to an issue thread!

**Comment by cmdcolin** [comment, marked as answer]
Hi there,
This was finally released in v2.9.0

You can use ?loc=GENENAME just like in jbrowse 1 if you have used the text-index command

Docs https://jbrowse.org/jb2/docs/urlparams/
